src/dotnet/Fable.JS/Fable.JS.fsproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="../../../paket-files/ncave/FCS/fcs/fcs-fable/fcs-fable.fsproj" />
+    <ProjectReference Include="../../../../FSharp.Compiler.Service_fable/fcs/fcs-fable/fcs-fable.fsproj" />
```

Comment:
Hmm, going back to using repos in parallel directories is convenient for local testing, but will prevent building the repl as an Appveyor artifact. Maybe leave a comment at least so we can use Paket dependencies again when we make everything work with Fable 2?

Reply:
Fixed.